### PR TITLE
Fix code style

### PR DESCRIPTION
**Pull request by JohnTitor**
It bumps up minimum supported version to 1.27.1 (for using `dyn` and `..=`).
I avoid using `#[rustfmt:skip]` because it's experimental in that version.

### CODE REVIEW

File: src/c_api.rs [hunk 1]
```diff
@@ -50,12 +50,11 @@ pub struct ExtensionBlock {
...
-//#define CONTINUE_EXT_FUNC_CODE    0x00    /* continuation subblock */
-//#define COMMENT_EXT_FUNC_CODE     0xfe    /* comment */
-//#define GRAPHICS_EXT_FUNC_CODE    0xf9    /* graphics control (GIF89) */
-//#define PLAINTEXT_EXT_FUNC_CODE   0x01    /* plaintext */
-//#define APPLICATION_EXT_FUNC_CODE 0xff    /* application block */
+    pub Function: c_int, //#define CONTINUE_EXT_FUNC_CODE    0x00    /* continuation subblock */
+                         //#define COMMENT_EXT_FUNC_CODE     0xfe    /* comment */
+                         //#define GRAPHICS_EXT_FUNC_CODE    0xf9    /* graphics control (GIF89) */
+                         //#define PLAINTEXT_EXT_FUNC_CODE   0x01    /* plaintext */
+                         //#define APPLICATION_EXT_FUNC_CODE 0xff    /* application block */
```
Comment (HeroicKatora) [inline src/c_api.rs]: Looks like the comments are supposed to be possible values defined for `Function` but no enum was created. Either way, that is the wrong formatting.

**Comment by HeroicKatora** [comment]
Since this is still a dependency of `image = 0.21` and there seems to be a bug fix waiting on review, I'd keep this in limbo for just a tiny bit to publish `0.10.3` and then apply this for a `0.11` where some other interface changes are also done.

File: src/c_api.rs (hunk 1 above)
Comment (JohnTitor) [reply src/c_api.rs]: Indeed, re-formatted.

**Comment by JohnTitor** [comment]
I think this PR is staled so closing for now. I'll re-submit another one if I find the time.

**Pull request closed without merging**